FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.jsx

```diff
@@ -60,8 +60,8 @@ const GeneralInfo = (props) => {
   const { profile: mainProfile, mutate: profileMutate } = useProfile();
   const [profile, setProfile] = useState(props.profile);
   const [formStateErr, setFormStateErr] = useState(ERRORS);
-  const [bannerImage, setBannerImage] = useState(props.profile.banner_image);
-  const [logoImage, setLogoImage] = useState(props.profile.logo_image);
+  const [bannerImage, setBannerImage] = useState(props.profile.banner?.path);
+  const [logoImage, setLogoImage] = useState(props.profile.logo?.path);
   const [bannerImageError, setBannerImageError] = useState(null);
   const [logoImageError, setLogoImageError] = useState(null);
   const [edrpouFieldError, setEdrpouFieldError] = useState(null);
```

Comment:
For complex state logic you may consider using  [useReducer](https://react.dev/reference/react/useReducer)